src/txn/txn.c

```diff
@@ -735,6 +735,10 @@ __wt_txn_config(WT_SESSION_IMPL *session, WT_CONF *conf)
         WT_ERR(__wti_txn_set_read_timestamp(session, read_ts));
     }
 
+    /* Claim transaction's prepared id */
```

Comment:
This is probably not correct but I'll address in my poc pr.

Reply:
Sure, this pr's main goal is to add the config option and doesnt actually do anything with the value yet. Thanks for noting it for future ref!